setup.py

```diff
@@ -14,7 +14,7 @@
                 'development in PyTorch.',
     author='Preferred Networks, Inc.',
     license='MIT License',
-    install_requires=['numpy', 'torch'],
+    install_requires=['numpy', 'torch', 'typing-extensions'],
```

Comment:
Better to add a version restriction (`>=3.10`)?

Reply:
Fixed.